### ISSUE TITLE
Acceptance tests failing

### ISSUE THREAD

**Issue by phil-davis** [owncloud/oauth2#159]
The overnight CI run had acceptance test failures. They were the same in both drone jobs. They are reproducible the same by making a PR. See example in PR #158 

```
--- Failed scenarios:

    /var/www/owncloud/apps/oauth2/tests/acceptance/features/webUIOauth2/newClientRegistration.feature:13
    /var/www/owncloud/apps/oauth2/tests/acceptance/features/webUIOauth2/obtainAccessToken.feature:18
    /var/www/owncloud/apps/oauth2/tests/acceptance/features/webUIOauth2/obtainAccessToken.feature:29
    /var/www/owncloud/apps/oauth2/tests/acceptance/features/webUIOauth2/revokeAccessToken.feature:12

18 scenarios (14 passed, 4 failed)
116 steps (112 passed, 4 failed)
4m47.61s (26.66Mb)
```

**Comment by phil-davis** [comment]
PR #160

**Comment by individual-it** [comment]
fixed by #160
